contracts/profit/src/state/mod.rs

```diff
@@ -36,13 +40,33 @@ const STATE: Item<'static, State> = Item::new("contract_state");
 
 type IcaConnector = dex::IcaConnector<OpenIca, ContractResult<DexResponse<Idle>>>;
 
+fn on_config_update(new_config: &Config, now: Timestamp) -> ContractResult<PlatformResponse> {
+    new_config
+        .time_alarms()
+        .setup_alarm(now + Duration::from_hours(new_config.cadence_hours()))
+        .map(PlatformResponse::messages_only)
+        .map_err(Into::into)
+}
+
```

Comment:
this is a copy/paste of `Idle::enter`, please remove

Reply:
Wasn't even an intentionally made. Will clean it up